File: background-js/src/Background.kt
import chrome.tabs.Tab

fun main() {
    chrome.runtime.onInstalled.addListener {
        console.log("Installed yeah!!")
        val tab = Tab {
            url = "https://google.com"
        }
        chrome.tabs.create(tab) {
            println("Tab created ${it.url}")
        }
    }
}

File: background.js
chrome.runtime.onInstalled.addListener(function (object) {
    chrome.tabs.create({url: "https://github.com/theapache64/source-pilot/blob/master/WELCOME.md#source-pilot"}, function (tab) {
        console.log("New tab launched with http://yoursite.com/");
    });
});